env/end-to-end_no_new_docker_images.sh

```diff
@@ -0,0 +1,2 @@
+export CBIOPORTAL_URL="https://rc.cbioportal.org"
```

Comment:
you prolly don't need this env now that you have a PR open?

Reply:
File was removed.